web_client/panels/DrawWidget.js

```diff
@@ -50,6 +53,16 @@ var DrawWidget = Panel.extend({
                 this._groups.get(this._style.id).save();
             }
         });
+        this.on('h:mouseover', (model) => {
```

Comment:
The `mouseover` and `randomevent` handlers are too slow.  Specifically, the `this.render()` call rerenders the entire draw panel widget when we really just want to toggle a class on a single element.  Instead of `this._skipRedraw = true; this.render();`, `this.$('.h-element[data-id="' + model.id + '"]').addClass('h-highlight-element');` would functionally do the same thing (although duplicating some logic that is in the pug template), and would be vastly faster.  Similarly, to turn off the highlighting, the code could be the same except removing the class.